pkg/cli/subcommands/health/command.go

```diff
@@ -52,10 +59,11 @@ func MakeCommand(globalParamsGetter func() GlobalParams) *cobra.Command {
 		},
 	}
 
+	cmd.Flags().IntVarP(&cliParams.timeout, "timeout", "t", 60, "timeout in second to query the Agent")
```

Comment:
@hush-hush 60 seconds seems like a really long time for a loopback transport timeout. Thoughts around reducing this a bit (20s? maybe something else?)?